Vagrantfile

```diff
@@ -244,6 +244,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                     end
                     formula_path
                 }
+                # write the minion file
+                minion_cfg = YAML.load_file("scripts/salt/minion.template")
+                more_base_paths = formula_paths.map{|p| "/vagrant/" + p + "/"}
+                minion_cfg['file_roots']['base'].insert(1, *more_base_paths)
```

Comment:
so this keeps inserting the depended-upon formulas in the place in the list where `builder-base-formula` was before, which is good